osu.Game.Rulesets.Rush/Objects/DualHit.cs

```diff
@@ -20,6 +23,10 @@ public override double StartTime
         public readonly DualHitPart Air = new DualHitPart { Lane = LanedHitLane.Air };
         public readonly DualHitPart Ground = new DualHitPart { Lane = LanedHitLane.Ground };
 
+        public override Judgement CreateJudgement() => new RushIgnoreJudgement();
```

Comment:
This removes it from the max combo calculation too, right?

Reply:
Yes